api/REST-API.md

```diff
@@ -1,90 +1,92 @@
-# Example Topic Outline (Replace with Title)
+# REST API
 
-### Projected Time
+### Projected Time- 1-1.5 hrs
 
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+- Lesson: 10 min
+- Guided Practice: 30 min
+- Independent Practice: 30 min
+- Check for Understanding: 5 min
 
 ### Prerequisites
 
 Here are links to lessons that should be completed before this lesson:
+- [API/JSON Lesson](/api/apis-and-json.md)
 
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+REST completely changed software engineering after 2000. This new approach to developing web projects and services was defined by Roy Fielding, father of the HTTP specification and one of the leading international authorities on everything to do with Network Architecture, in his dissertation entitled "Architectural Styles and the Design of Network-based Software Architectures". In the field of APIs, REST (Representational State Transfer) is today the "be all and end all" in service app development.
+Today there are no projects or applications that don't have a REST API for the creation of professional services based on this software. Twitter, YouTube, Facebook identification systems… hundreds of companies generate business thanks to REST and REST APIs. Without them any horizontal growth would be practically impossible. This is because REST is the most logical, efficient and widespread standard in the creation of APIs for Internet services.
```

Comment:
There's a underscore(_) after the word systems.

Reply:
I think it’s an ellipses (...)

Reply:
Yeah, it looks like underscore in that small font...I checked it now on the big screen.
Sorry for the inconvenience.